contrib/imgui/imgui_draw.cpp

```diff
@@ -1800,15 +1817,31 @@ void ImFont::AddRemapChar(ImWchar dst, ImWchar src, bool overwrite_dst)
     IndexXAdvance[dst] = (src < index_size) ? IndexXAdvance.Data[src] : 1.0f;
 }
 
-const ImFont::Glyph* ImFont::FindGlyph(unsigned short c) const
+const ImFont::Glyph* ImFont::FindGlyph(unsigned short c, bool report_missing) const
 {
-    if (c < IndexLookup.Size)
+	if (c < IndexLookup.Size)
```

Comment:
What has happened here is that this contrib file is space indented (4 char wide), but your editor changed this line to tab.

Actually, this file is a bit of a mess. I suspect most is space indented, but stuff you edited is tabs.
 
Doens't get better by github by default showing tabs as 3 chars wide.

I vote: do nothing.